src/crepe/facade/SDLContext.h

```diff
@@ -53,13 +77,40 @@ class SDLContext {
 	SDLContext & operator=(SDLContext &&) = delete;
 
 private:
-	//! will only use handle_events
-	friend class LoopManager;
+	//! will only use get_events
+	friend class InputSystem;
+	/**
+	 * @brief Retrieves a list of all events from the SDL context.
+	 * 
+	 * This method retrieves all the events from the SDL context that are currently
+	 * available. It is primarily used by the InputSystem to process various
+	 * input events such as mouse clicks, mouse movements, and keyboard presses.
+	 * 
+	 * @return A vector of `SDLContext::EventData` containing the events.
```

Comment:
Ik kan al zien dat het een vector van eventdata is aan de return type. Hier zou ik liever
```
\return Events that occurred since last call to `get_events()`
```
of iets dergelijks terug zien.